homeassistant/components/bluetooth_tracker/device_tracker.py

```diff
@@ -65,15 +65,15 @@ def discover_devices(device_id: int) -> List[Tuple[str, str]]:
     return result
 
 
-def see_device(
+async def see_device(
     hass: HomeAssistantType, async_see, mac: str, device_name: str, rssi=None
 ) -> None:
     """Mark a device as seen."""
     attributes = {}
     if rssi is not None:
         attributes["rssi"] = rssi
 
-    hass.async_create_task(
+    await hass.async_create_task(
```

Comment:
We don't need to create a task here. We can just await the coroutine directly.

Reply:
Thanks for spotting this!